pom.xml

```diff
@@ -499,6 +499,11 @@
             <version>0.8.13</version>
             <scope>test</scope>
         </dependency>
+        <dependency>
+            <groupId>org.springframework</groupId>
```

Comment:
1. why do you need this dependency. which code relies on that?
2. why do you specify that version on POM? our POM inherits dependency management from spring boot parent BOM so the correct version is already defined there

Reply:
this dependency is for Spring cacheable

Reply:
I had build errors, so I had to use <version>5.2.2.RELEASE</version>

Reply:
Strange, that should work out of the box. I need to check on my local

Reply:
<dependency>
    Did you try following?

<groupId>org.springframework.boot</groupId>
    <artifactId>spring-boot-starter-cache</artifactId>
</dependency>

Reply:
No I will try Spring-boot.

Reply:
Yes spring-boot-starter-cache works fine